src/main/webapp/app/store/WindowStore.ts

```diff
@@ -11,6 +11,7 @@ export interface IWindowSize {
 class WindowStore {
   @observable size: IWindowSize;
   public recaptchaRef: any;
+  @observable recaptchaVerified: boolean;
```

Comment:
The goal is to verify one time and works for the rest of the session?

Reply:
Yes.